<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeDataTimerEmbeddedWidget</class>
 <widget class="QWidget" name="NodeDataTimerEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="timeInputLayout">
     <item>
      <widget class="QSpinBox" name="mpSecondSpinbox"/>
     </item>
     <item>
      <widget class="QLabel" name="mpSecondLabel">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mpMillisecondSpinbox"/>
     </item>
     <item>
      <widget class="QLabel" name="mpMillisecondLabel">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pfLayout">
     <item>
      <widget class="QComboBox" name="mpPFComboBox">
       <item>
        <property name="text">
         <string>Period</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Frequency</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpPFLabel">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpPFUnitLabel">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="pfSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpStartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="remainingLayout">
     <item>
      <widget class="QLabel" name="mpRemainingTextLabel">
       <property name="text">
        <string>Time Remaining:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpRemainingLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mpRemainingUnitLabel">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="remainingSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpStopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="resetLayout">
     <item>
      <spacer name="resetSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpResetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
